tests/unit_tests/tools/test_linker.py

```diff
@@ -124,7 +124,7 @@ def test_get_lib_flags_unknown(stub_c_compiler: CCompiler) -> None:
     linker = Linker(compiler=stub_c_compiler)
     with raises(RuntimeError) as err:
         linker.get_lib_flags("unknown")
-    assert str(err.value) == "Unknown library name: 'unknown'"
+    assert str(err.value) == "unknown library unknown"
```

Comment:
This might benefit from a redrafting. Initial upper-case and maybe quote marks around library name.